prometheus/src/cluster_metrics.rs

```diff
@@ -52,5 +81,60 @@ pub fn write_cluster_metrics<W: io::Write>(
         },
     )?;
 
+    // Vote accounts information
+    for vote_account in vote_accounts.iter() {
+        write_metric(
+            out,
+            &MetricFamily {
+                name: "solana_node_last_vote_slot",
+                help:
+                    "The voted-on slot of the validator's last vote that got included in the chain",
+                type_: "gauge",
+                metrics: banks_with_commitments.for_each_commitment(|bank| {
+                    let vote_info = get_vote_state(bank, vote_account)?;
+                    Some(
+                        Metric::new(vote_info.last_vote)
+                            .with_label("identity_account", identity_pubkey.to_string())
+                            .with_label("vote_account", vote_account.to_string()),
+                    )
+                }),
+            },
+        )?;
+
+        write_metric(
+            out,
+            &MetricFamily {
+                name: "solana_node_vote_balance_sol",
+                help: "The current node's vote account balance",
```

Comment:
```suggestion
                help: "The balance of the vote account at the given address",
```
It’s not just the current node any more!